package.json

```diff
@@ -40,10 +40,13 @@
     }
   },
   "dependencies": {
+    "@reach/dialog": "^0.7.0",
     "dayjs": "^1.8.16",
     "js-cookie": "^2.2.1",
     "lazysizes": "^5.1.2",
+    "react-popper": "^1.3.7",
     "react-select": "^3.0.8",
+    "react-spring": "^8.0.27",
```

Comment:
https://www.react-spring.io/

Very well supported Lib that makes animations extremely easy, treeshakeable and relatively small. Thinking about future animations on EDS components that this could make a lot easier.

Reply:
+1. I remember doing a deep dive on React animation a couple years ago and I recall this being one of the good ones. Curious what's planned for it. Did you look into other libs?

Reply:
Non-blocking

> 10.7KB (web), 9.7KB (react-native) or 6.9KB (/universal export). The size will ultimately depend on your build-chain and can decrease with tree-shaking. For instance, just importing useSpring without color-interpolations (using the /universal export) you will get 4.7KB.

I'd like to keep an eye out for how we use this and better understand it. Sounds like it's likely a good one, but let's try to sync up so I can stay in the loop.

Reply:
Removed this lib for now